2024-08-13 12:58:33,750 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:06:22,695 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:06:26,916 ERROR: Exception on /register [GET] [in /Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/app.py:838]
Traceback (most recent call last):
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/app/routes.py", line 74, in register
    return render_template('register.html', title='Register', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/ezhong/Documents/microblog/app/templates/register.html", line 2, in top-level template code
    {% import 'bootstrap_wtf.html' as wtf %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ezhong/Documents/microblog/venv/lib/python3.11/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: bootstrap_wtf.html
2024-08-13 13:09:21,050 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:11:38,380 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:11:49,491 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:11:58,942 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:13:07,974 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:25:42,156 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:26:16,343 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
2024-08-13 13:27:15,149 INFO: Microblog startup [in /Users/ezhong/Documents/microblog/app/__init__.py:44]
